### PR TITLE
[Model] Create Person Model

### PR DESCRIPTION
**Pull request by raphaelsattler**
Created Person models with validations and its tests.

### CODE REVIEW

File: app/models/address.rb [hunk 1]
```diff
@@ -8,5 +8,7 @@ class Address < ApplicationRecord
   validates :country, presence: true
   validates :active, inclusion: { in: [true, false] }
 
+  belongs_to :person
   belongs_to :addressable, polymorphic: true
+  belongs_to :person
```
Comment (kaiomagalhaes) [inline app/models/address.rb:12]: this is duplicated

File: app/models/buy_intention.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class BuyIntention < ApplicationRecord
+  validates :name, presence: true, uniqueness: true
```
Comment (kaiomagalhaes) [inline app/models/buy_intention.rb:2]: why is the name unique?

File: app/models/profile.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+class Profile < ApplicationRecord
+  validates :name, presence: true, uniqueness: true
```
Comment (kaiomagalhaes) [inline app/models/profile.rb:2]: Why does a Profile has an unique name?

File: app/models/telephone.rb [hunk 4]
```diff
@@ -5,5 +5,7 @@ class Telephone < ApplicationRecord
   validates :telephone_number, presence: true, length: { maximum: 10 }
   validates :active, inclusion: { in: [true, false] }
 
+  belongs_to :person
   belongs_to :telephoneable, polymorphic: true
+  belongs_to :person
```
Comment (kaiomagalhaes) [inline app/models/telephone.rb:9]: this is duplicated

File: app/models/address.rb (hunk 1 above)
Comment (RodrigoNM) [inline app/models/address.rb:12]: Why this class belongs to person twice?

File: spec/factories/person.rb [hunk 5]
```diff
@@ -0,0 +1,14 @@
+FactoryBot.define do
+  factory :person do
+    profile { association(:profile) }
+    buy_intention { BuyIntention.first || association(:buy_intention) }
```
Comment (kaiomagalhaes) [inline spec/factories/person.rb]: We should avoid this approach

**Review by kaiomagalhaes — CHANGES_REQUESTED** [review]

File: app/models/telephone.rb (hunk 4 above)
Comment (RodrigoNM) [inline app/models/telephone.rb:9]: This class also belongs twice to person?

File: config/environments/production.rb [hunk 6]
```diff
@@ -54,7 +54,7 @@
   config.log_level = :debug
 
   # Prepend all log lines with the following tags.
-  config.log_tags = [:request_id]
+  config.log_tags = [ :request_id ]
```
Comment (RodrigoNM) [inline config/environments/production.rb]: I'm not sure, but i think theres no white space before brackets.

File: spec/models/person_spec.rb [hunk 7]
```diff
@@ -0,0 +1,41 @@
+require "rails_helper"
+
+RSpec.describe Person, type: :model do
+  it {  is_expected.to have_many(:telephones) }
+
```
Comment (RodrigoNM) [inline spec/models/person_spec.rb:5]: I think that it's better if you remove these blank lines between each it.

File: spec/models/profile_spec.rb [hunk 8]
```diff
@@ -0,0 +1,13 @@
+require "rails_helper"
+
+RSpec.describe Profile, type: :model do
+  let(:profile) { create(:profile) }
+
```
Comment (RodrigoNM) [inline spec/models/profile_spec.rb:5]: Same as the comment above, it's better if you remove these blank lines.

File: spec/models/telephone_spec.rb [hunk 9]
```diff
@@ -3,6 +3,8 @@
 RSpec.describe Telephone, type: :model do
   let(:telephone) { create(:telephone) }
 
+  it { is_expected.to belong_to(:person) }
+
```
Comment (RodrigoNM) [inline spec/models/telephone_spec.rb]: Unnecessary blank lines.

**Review by kaiomagalhaes — DISMISSED** [review]

File: app/models/profile.rb (hunk 3 above)
Comment (raphaelsattler) [reply app/models/profile.rb:2]: Because it's a person model classification.

File: app/models/buy_intention.rb (hunk 2 above)
Comment (raphaelsattler) [reply app/models/buy_intention.rb:2]: Because it's a person model classification.

**Review by kaiomagalhaes — APPROVED** [review]

**Review by RodrigoNM — APPROVED** [review]

**Review by Heirian — APPROVED** [review]

**Pull request merged**